src/components/BatchAllocations/hooks/useBatchAllocationLogs/useBatchAllocationLogs.js

```diff
@@ -20,9 +23,14 @@ export const useBatchAllocationLogs = (options = {}) => {
   const ky = useOkapiKy({ tenant: tenantId });
   const [namespace] = useNamespace({ key: 'fund-update-logs' });
 
+  const { search } = useLocation();
```

Comment:
please provide `filters` to the hook instead (`useBatchAllocationLogs = ({ filters }, options = {}) => `)